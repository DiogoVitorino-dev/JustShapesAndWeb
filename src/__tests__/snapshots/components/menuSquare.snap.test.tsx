import { render } from "@testing-library/react-native";

import MenuSquare from "@/components/menu/menuSquare";

describe("Menu Square - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<MenuSquare />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
