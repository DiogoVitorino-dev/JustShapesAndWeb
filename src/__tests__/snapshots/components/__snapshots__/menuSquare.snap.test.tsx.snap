// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Square - Snapshot component test should render correctly 1`] = `
<View
  collapsable={false}
  jestAnimatedStyle={
    {
      "value": {
        "backgroundColor": "rgb(8, 94, 63)",
        "borderRadius": 3,
        "height": 217.91575375955455,
        "left": 600,
        "opacity": 0.4,
        "position": "absolute",
        "top": 1068,
        "transform": [
          {
            "rotate": "216deg",
          },
          {
            "scale": 1.0010073205534684,
          },
        ],
        "width": 123.91575375955455,
      },
    }
  }
  style={
    {
      "backgroundColor": "rgb(8, 94, 63)",
      "borderRadius": 3,
      "height": 214,
      "left": 600,
      "opacity": 0.4,
      "position": "absolute",
      "top": 1068,
      "transform": [
        {
          "rotate": "216deg",
        },
        {
          "scale": 1,
        },
      ],
      "width": 120,
    }
  }
/>
`;
